docs/content/Getting-Started/quickstart.md

```diff
@@ -87,15 +87,31 @@ workload and summarization aspirations removed.
 Create the first edge cluster:
 
 ```shell
-kind create cluster --name florin --config  kubestellar/examples/florin-config.yaml
+kind create cluster --name florin --config - <<EOF
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  extraPortMappings:
+  - containerPort: 8081
+    hostPort: 8081
+EOF
 ```  
 
 Note: if you already have a cluster named 'florin' from a previous exercise of KubeStellar, please delete the florin cluster ('kind delete cluster --name florin') and create it using the instruction above.
 
 Create the second edge cluster:
 
 ```shell
-kind create cluster --name guilder --config  kubestellar/examples/guilder-config.yaml
+kind create cluster --name guilder --config - <<EOF
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  extraPortMappings:
+  - containerPort: 8081
+    hostPort: 8096
+EOF
```

Comment:
The file in `examples` has two port mappings here, which supports users who go on to the extended example.  If the second port mapping is excluded here then users can not go on to the extended example.

Reply:
Fixed. I added the second port mapping for the guilder cluster